src/pool.c

```diff
@@ -805,6 +805,29 @@ worker_list(char *list_start, char *list_end, const char *address)
     pthread_rwlock_unlock(&rwlock_acc);
 }
 
+void
+worker_hr(double *avg, const char *address, const char *rig_id)
+{
+    account_t *account = NULL;
+    if (!address || !rig_id || !*address || !*rig_id)
+        return;
+    pthread_rwlock_rdlock(&rwlock_acc);
+    HASH_FIND_STR(accounts, address, account);
+    if (!account)
+        goto bail;
+    client_t *c = (client_t*)gbag_first(bag_clients);
+    while ((c = gbag_next(bag_clients, 0)))
+    {
+        if (strncmp(c->address, address, ADDRESS_MAX) == 0 && strncmp(c->rig_id, rig_id, MAX_RIG_ID) == 0)
```

Comment:
Can you wrap at 80, like everywhere is please. Trying to keep consistent formatting..

Reply:
added an 80 ruler to my editor, will wrap at this marker.

Reply:
Wrapping at 80 would look like
```bash
  if (strncmp(c->address, address, ADDRESS_MAX) == 0 && strncmp(c->rig_id, 
      rig_id, MAX_RIG_ID) == 0)
```
While wrapping at a logical point could look like:
```bash
  if (strncmp(c->address, address, ADDRESS_MAX) == 0 && 
      strncmp(c->rig_id, rig_id, MAX_RIG_ID) == 0)
```

I would definitely prefer the second.

Reply:
Exactly. As I mentioned "Can you wrap at 80, _like everywhere is please_", which wraps like @trasherdk commented.

Reply:
I will perform the wrapping as suggested by trasherdk.    When I said I was placing the marker, I did not mean I would wrap directly at 80.   Just so that I know I need to wrap the line when it breaks the ruler mark.

New push eta ~end of week.